File: public/usage-examples/utilities/trim-1-example.py
from splashkit import *

text = "   Whitespace is sneaky!   "

write_line(f"Original string with sneaky spaces: '{text}'")
write_line("Let's get rid of those pesky spaces...")

# Trim spaces from the start and end
trimmed = trim(text)

write_line(f"Trimmed string: '{trimmed}'")
write_line("Aha! Much better without those sneaky spaces!")

File: public/usage-examples/utilities/trim/trim-1-example.cpp
#include "splashkit.h"

int main()
{
    string text = "   Whitespace is sneaky!   ";

    write_line("Original string with sneaky spaces: '" + text + "'");
    write_line("Let's get rid of those pesky spaces...");

    // Trim spaces from the start and end
    string trimmed = trim(text);

    write_line("Trimmed string: '" + trimmed + "'");
    write_line("Aha! Much better without those sneaky spaces!");

    return 0;
}
